lib/gman_client.rb

```diff
@@ -8,4 +8,14 @@ def self.drivers
     respond = request.api.v1.drivers.get
     respond.map(&:to_h)
   end
+  def self.driver_commission_histories
+    request = Blanket.wrap("http://localhost:3000", :extension => :json)
```

Comment:
How about passing the url into the gman client calls as another parameter?

Or you could new up a client like...

``` ruby
class GmanClient
  def initialize(host = nil)
    fail ArgumentError.new('host is required') if host.nil?

    @host = host
  end

  def drivers
    response = request.api.v1.drivers.get
    response.map(&:to_h)
  end

  # other API methods

  def request
    Blanket.wrap(@host, :extension => :json)
  end
end

```

And use like

```ruby
  client = GmanClient.new('http://gman-services.dev')
  drivers = client.drivers
```